css/styles.reforging.css

```
@@ -0,0 +1,469 @@
+  font-size: 0;
+  color: var(--text);
+  opacity: 0.78;
+  line-height: 1;
+  transition: color 0.2s ease, background-color 0.2s ease, opacity 0.2s ease;
+}
+
+.reforging-set-chevron::before {
+  content: "";
+  width: 9px;
+  height: 9px;
+  border-right: 2px solid currentColor;
```

Stylelint (17.3.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)